pkg/kubectl/resource/builder.go

```diff
@@ -360,13 +396,33 @@ func hasCombinedTypeArgs(args []string) (bool, error) {
 	switch {
 	case hasSlash > 0 && hasSlash == len(args):
 		return true, nil
-	case hasSlash > 0 && hasSlash != len(args):
-		return true, fmt.Errorf("when passing arguments in resource/name form, all arguments must include the resource")
```

Comment:
Sorry, I confused this with other stuffs...
I changed its behavior so now `a/b c` will be seen as "find me the resource b in group a named c"...